website/docs/fr-FR/quickstart.md

```diff
@@ -33,42 +33,137 @@ app.mount('#app')
 
 L'exemple ci-dessus importe Element Plus entièrement. Notez que les fichiers CSS doivent être importés séparément.
 
-#### À la demande
+#### On demand in Vue CLI
 
-Grâce au [babel-plugin-import](https://github.com/ant-design/babel-plugin-import), nous pouvons importer uniquement les composants désirés, rendant ainsi le projet plus léger.
+With the help of [babel-plugin-import](https://github.com/ant-design/babel-plugin-import), we can import components we actually need, making the project smaller than otherwise.
 
-Tout d'abord, installez babel-plugin-import:
+Firstly，install babel-plugin-import:
 
 ```bash
-npm install babel-plugin-import -D
+$ npm install babel-plugin-import -D
 ```
 
-Puis éditez babel.config.js:
+or if you use `Yarn` as package manager
```

Comment:
Don't change these text `Puis éditez babel.config.js` means "then edit babel.config.js". `Tout d'abord, installez` means "first of all, install"